examples/flask-web-app/DEMO.md

```diff
@@ -3,12 +3,26 @@
 # Preparation
 * Before the demo:
     * clean out all Docker containers and images, except Phusion
-    * Rebuild: `flyingcloud sysbase; flyingcloud pybase; flyingcloud opencv; flyingcloud app`
-
+    * Rebuild:
+        ```bash
+  flyingcloud --no-push --no-pull sysbase
+  flyingcloud --no-push --no-pull pybase
+  flyingcloud --no-push --no-pull opencv
+  flyingcloud --no-push --no-pull opencv
```

Comment:
I fixed the backquotes on master after I did this PR; will fix the duplicate opencv line later. Thanks for catching that.